frontend/device/devicelistener.lua

```diff
@@ -51,24 +51,38 @@ end
 function DeviceListener:onShowWarmth(value)
     local powerd = Device:getPowerDevice()
     if powerd.fl_warmth ~= nil then
-        UIManager:show(Notification:new{
-            text = T(_("Warmth set to %1."), powerd.fl_warmth),
+        if powerd.fl_warmth_max == nil then
+            UIManager:show(Notification:new{
+                text = T(_("Warmth set to %1."), math.floor(powerd.fl_warmth)),
+                timeout = 1.0,
+            })
+        else
+            UIManager:show(Notification:new{
+            text = T(_("Warmth set to %1."), math.floor(powerd.fl_warmth/100*powerd.fl_warmth_max)),
             timeout = 1.0,
-        })
+            })
+        end
     end
     return true
 end
 
 -- frontlight controller
 if Device:hasFrontlight() then
 
-    -- direction +1 - increase frontlight
-    -- direction -1 - decrease frontlight
-    function DeviceListener:onChangeFlIntensity(ges, direction)
-        local powerd = Device:getPowerDevice()
+    local function calculateGestureDeltaIntervall(ges, direction, min, max)
```

Comment:
interval has only one l, but I find this an odd function name. It seems to be about the delta of distance moved; where does an interval come in?
```suggestion
    local function calculateGestureDeltaInterval(ges, direction, min, max)
```

Reply:
OK, double 'l' is German. I think Interval is not so bad, as the function calculates the delta from the gesture move width and maps this to min-max.
Which name would you suggest?

Reply:
Since you seem to be using them as a tautology, just delta (e.g., calculateGestureDelta).

Reply:
OK, I will use that.